Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -320,17 +320,34 @@ The dynamic analysis of this section comprises of two parts: the actual license
 
 It need to be validated whether the copyrights of the licenses have been adhered to. This often means that the application should have an `about` or `EULA` section in which the copy-right statements are noted as required by the license of the third party library.
 
-When no source-code is available for library analysis, you can find some of the frameworks being used with otool and MobSF.
-After you obtain the library and Clutched it (e.g. removed the DRM), you can run oTool with the root of the application's directory:
+##### Listing Application Libraries
+
+When performing app analysis, it is important to also analyze the app dependencies (usually in form of libraries or so-called iOS Frameworks) and ensure that they don't contain any vulnerabilities. Even when you don't have the source code, you can still identify some of the app dependiencies using tools like [objection](https://github.com/sensepost/objection), [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF) or otool. Objection is the recommended tool, it gives the most accurate results and it is easy to use.
```

Comment:
Corrected word: dependencies

Merged below sentence here. I'd remove then lines 326-329 since we don't need a subsection anymore.

```suggestion
When performing app analysis, it is important to also analyze the app dependencies (usually in form of libraries or so-called iOS Frameworks) and ensure that they don't contain any vulnerabilities. Even when you don't have the source code, you can still identify some of the app dependencies using tools like [objection](https://github.com/sensepost/objection), [MobSF](https://github.com/MobSF/Mobile-Security-Framework-MobSF) or otool. Objection is the recommended tool, since it provides the most accurate results and it is easy to use. It contains a module to work with iOS Bundles, which offers two commands: `list_bundles` and `list_frameworks`.
```